### ISSUE TITLE
Code snippet does not base64 encode the basic authentication header

### ISSUE THREAD

**Issue by worawatt** [Huachao/vscode-restclient#741]
When creating code snippet from a request with basic authentication using user:password format, the Authorization in the snippet is not encoded with base64.

For example
```
GET https://httpbin.org/basic-auth/user/passwd HTTP/1.1
Authorization: Basic user:passwd
```
Result:
```
GET /basic-auth/user/passwd HTTP/1.1
User-Agent: vscode-restclient
Authorization: Basic user:passwd
Host: httpbin.org
```

But using user password format works correctly.
```
GET https://httpbin.org/basic-auth/user/passwd HTTP/1.1
Authorization: Basic user passwd
```
Result:
```
GET /basic-auth/user/passwd HTTP/1.1
User-Agent: vscode-restclient
Authorization: Basic dXNlcjpwYXNzd2Q=
Host: httpbin.org
```

Maybe because of this line where it splits the user and password using a space only:
https://github.com/Huachao/vscode-restclient/blob/c7e783b6a20826140b18d30ca695199e3dd08098/src/controllers/codeSnippetController.ts#L194

**Comment by Huachao** [comment]
@worawatt nice catch, I will publish this fix in the next release.

**Comment by Huachao** [comment]
@worawatt this has been fixed in the latest version 0.24.5 and you can have a try

**Comment by worawatt** [comment]
It works.
Thank you so much @Huachao.
This plugin is awesome, and I've always suggested it to my colleagues.
